### ISSUE TITLE
Minimize Group

### ISSUE THREAD

**Issue by nickclark2016** [vurtun/nuklear#541]
I have some code that looks like: 
```cpp
if (nk_begin(Ctx, Title_.c_str(), nk_rect(X, Y, Width, Height, Flags))
{
    nk_layout_row_dynamic(Ctx, 360, 1);
    if (nk_group_begin(Ctx, "Test", NK_WINDOW_MINIMIZABLE | NK_WINDOW_TITLE | 
        NK_WINDOW_BORDER))
    {
        ...
        nk_group_end(Ctx);
    }
}
nk_end(Ctx);
```

It appears to render correctly, but when I minimize the window, `nk_group_scrolled_end(struct nk_context*)` crashes on `NK_ASSERT(g->parent)`.  Am I doing something wrong, or is there a bug?

**Comment by griebd** [comment]
not sure, bu have you tried moving `nk_group_end(Ctx);` outside of `if (nk_group_begin`?

**Comment by nickclark2016** [comment]
Doesn't make a difference if I call it inside or outside of the `if` block.

**Comment by nickclark2016** [comment]
*poke*

**Comment by greje656** [comment]
Yup, I have the same problem. It looks as if groups can't be minimized. Same assert

**Comment by vurtun** [comment]
Hey @nickclark2016, @greje656,
is there a special reason for wanting a minimizable group instead of using a tree node?

**Comment by nickclark2016** [comment]
To be honest, I can't remember for myself (it's been a while and I haven't worked on my engine UI recently).  However, I do remember that there was a reason.

**Comment by greje656** [comment]
I just liked the look of groups better than a tree node :)
![groups](https://user-images.githubusercontent.com/1680603/37006841-4d542c84-20a8-11e8-8b5e-64656aaa54ff.jpg)

**Comment by nickclark2016** [comment]
Is there any update on this issue?  From what I can tell, I still cannot minimize groups.  The sole reason I need it is because I like the way it looks more than tree nodes.

**Comment by kg** [comment]
I have this problem and also can't get tree nodes to collapse, so it would be cool to get a fix for it :(
